llvm/test/Transforms/Util/flatten-cfg.ll

```diff
@@ -1,5 +1,5 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 2
-; RUN: opt -passes=flatten-cfg -S < %s | FileCheck %s
+; RUN: opt -passes='require<aa>,flatten-cfg' -S < %s | FileCheck %s
```

Comment:
Shouldn't be needed? It's not an optional dependency at the PM level: `&AM.getResult<AAManager>(F)`